src/components/Patients/PatientTable.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { Table } from 'semantic-ui-react'
+import { Container, Segment } from "semantic-ui-react";
+import styled from "styled-components";
+
+import PatientRow from "components/Patients/PatientRow";
+
+const StyledContainer = styled(Container)`
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  margin: 1rem;
+`;
+
+const PatientTable = ({ filters, onApply, count, nextPage, totalPages, rows, error }) => {
+
+  if (error) {
+    return (
+      <StyledContainer>
+        <Segment inverted color="red" secondary>
+          {error}
+        </Segment>
+      </StyledContainer>
+    );
+  }
+
+  if (!rows) {
+    return null;
+  }
+
```

Comment:
You can add check if `rows.length === 0` and return `<Message>` component from semantic with message: `Patients not found`